cogs/tiles.py

```diff
@@ -80,32 +91,91 @@ def random_color():
                 raw_transparent = obj.tile.raw_transparent
                 tile = QudTile(filename, colorstring, colors[0], colors[1], qudname,
                                raw_transparent)
-            if smalltile:
+            if gif is not None:
+                data = GifHelper.get_bytesio(gif)
+            elif smalltile:
                 data = tile.get_bytesio()
             else:
                 data = tile.get_big_bytesio()
             data.seek(0)
-            msg = f"`{obj.name}` (display name: '{obj.displayname}'):"
-            return await ctx.send(msg, file=File(fp=data, filename=f'{obj.displayname}.png'))
+            msg += f"`{obj.name}` (display name: '{obj.displayname}'):"
+            if not smalltile and not gif and TileAnimator(obj).has_gif:
+                msg += f"\nThis tile can be animated (`?animate {obj.name}`)"
+            ext = '.png' if gif is None else '.gif'
+            return await ctx.send(msg, file=File(fp=data, filename=f'{obj.displayname}{ext}'))
         else:
             msg = f"Sorry, the Qud blueprint `{obj.name}` (display name: '{obj.displayname}')" \
                   " doesn't have a tile."
             await ctx.send(msg)
 
+    @command()
+    async def tile(self, ctx: Context, *args):
+        """Send the tile for the named Qud blueprint.
+
+        Supported command formats:
+          ?tile <object>
```

Comment:
I like this better than the old help text.

Reply:
👍